src/main/java/core/basesyntax/suppliers/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.suppliers;
+
+import core.basesyntax.enums.Color;
+import java.util.Random;
+
+public class ColorSupplier {
+    private final int index = new Random().nextInt(Color.values().length);
```

Comment:
random should be private final class-level field
https://github.com/mate-academy/jv-oop-advanced/blob/master/checklist.md#think-about-which-variables-should-be-local-in-the-method-and-which-should-be-class-level